newton/_src/sim/ik/ik_common.py

```
@@ -0,0 +1,129 @@
+            model.body_com,
+        ],
+        outputs=[body_q, body_qd],
+        device=model.device,
+    )
+
+
+@wp.kernel
+def _fk_accum(
+    joint_parent: wp.array1d(dtype=wp.int32),
+    X_local: wp.array2d(dtype=wp.transform),
+    joint_count: int,
```

Ruff (0.13.1)

Unused function argument: `joint_count`

(ARG001)